doc/nrf/libraries/modem/lte_lc.rst

```diff
@@ -19,6 +19,8 @@ Configuration
 *************
 
 To enable the library, set the Kconfig option :kconfig:option:`CONFIG_LTE_LINK_CONTROL` to ``y`` in the project configuration file :file:`prj.conf`.
+By default the library supports only network registration and connection status functionalities.
```

Comment:
Based on the changes, the default functionality seem to include also PSM (`lte_lc_psm_*()`) and periodic search (`lte_lc_periodic_search_*()`).
However, I think we should think about configuration for these as well. Especially for rarely used periodic search which should be behind configurations.